SQL/Archive/19.1/2018-02-20_remove_duplicate_value_from_Config.sql

```diff
@@ -0,0 +1,10 @@
+/*  delete duplicate ConfigID-value pairs */
```

Comment:
should this patch maybe be put in one of the new directories, either `Cleanup_patches` or `New_patches`?

Reply:
We don't have the folder as you said.
So, the best way is put it in the patch.